avocado/utils/linux_modules.py

```diff
@@ -153,3 +156,29 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+def check_kernel_config(config_name):
+    """
+    Method : pass a config option and it return
+             -config not  avilable
+             -config as in  loadable
+             -config already loaded
+    """
+
+    kernel_version = platform.uname()[2]
+
+    config_output = process.system_output(
+        'cat /boot/config-' + kernel_version +
+        '| grep -i --color=never ' + config_name, shell=True)
+
+    if "=" not in config_output:
+        return 'Config is not set'
```

Comment:
@ldoktor  sure , thanks for helping on this , will raise new PR with suggested changes 

Regards
Praveen